### PR TITLE
adding a new context from Minerva

### PR DESCRIPTION
**Pull request by dougli1sqrd**

### CODE REVIEW

**Comment by dougli1sqrd** [comment]
@jmcmurry Should we leave that as a different PR? All of these changes were for merely adding the minerva context. Along the way I had to fix travis by making python versions of the json manipulation scripts that were in perl, and then when I ran `make` it made the other changes. It looks like the files were already using identifiers.org?

File: registry/idot_context.jsonld [hunk 1]
```diff
@@ -261,7 +271,7 @@
     "MESH.2012": "http://identifiers.org/mesh.2012/",
```
Comment (jmcmurry) [inline registry/idot_context.jsonld]: is the 2012 bit really required for our purposes? Are we in a mixed version environment?

Comment (cmungall) [reply registry/idot_context.jsonld]: This is a direct transform of the idot registry. The role of this repo is to be a clearinghouse, from where intelligent decisions about what to mix and match can be made

**Comment by cmungall** [comment]
Yes, this is orthogonal to any long term decisions about identifiers, this is just getting the minerva context we use in Noctua registered

**Pull request merged**